### ISSUE TITLE
correct placement of alt text header in View Availability table

### ISSUE THREAD

**Issue by Smullz622** [psu-libraries/psulib_blacklight#1392]
In the View Availability table, there's an alt text header 'where to find this item in the library', but it's not showing up at the top like a proper header. Instead it reads in the middle of the table after the column headers.
